### PR TITLE
Update AddCommand to add Patients

### LINKED ISSUE #14: As a ward admin processing patients, I want to add and delete a patient
So that I can inpro and outpro patients as they are hospitalised and discharged

### PR DESCRIPTION
**Pull request by pzhengze**
Updated Parser to understand PID.
Updated AddCommand to add Patients.
Updated UI to display PID.
Updated Storage to understand PID.
Original delete still on patients no need to update.

This will close #14 if merged

### CODE REVIEW

**Comment by euph00** [comment]
Unfortunately need to resolve checkstyle issues
Also, the Model interface is about to change quite drastically wrt to switching from person objects to wards/patients/staff etc so a lot of things are about to break everywhere, since the model section is going through changes

**Comment by pzhengze** [comment]
What changes are you making with model exactly? We should minimise the changes so that we don't need to do so much work.

**Comment by euph00** [comment]
> What changes are you making with model exactly? We should minimise the changes so that we don't need to do so much work.

Changes to the API to support Patient and Staff objects with the relevant CRUD features for them, and containing them in Ward objects. Patientist will now hold a WardList object that contains 1..* Ward objects, and API supporting CRUD for this Wards feature will be included too.

Will open PR soon to show diffs in the API, but breaking changes are unavoidable if we are to implement wards/patients/staff classes

**Comment by pzhengze** [comment]
I see I'll close this pr until you make your pr then.

File: src/main/java/seedu/patientist/logic/parser/AddCommandParser.java [hunk 1]
```diff
@@ -31,22 +37,32 @@ public class AddCommandParser implements Parser<AddCommand> {
...
+        tagList.addAll(ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG)));
 
-        Person person = new Person(name, phone, email, address, tagList);
+        if (argMultimap.getValue(PREFIX_STATUS).isPresent()) {
+            PatientStatusDetails patientStatusDetails =
+                    ParserUtil.parseStatus(argMultimap.getValue(PREFIX_STATUS).get());
+            Patient patient = new Patient(idNumber, name, phone, email, address, patientStatusDetails, tagList);
 
-        return new AddCommand(person);
+            return new AddCommand(patient);
```
Comment (lukkesreysandeur) [inline src/main/java/seedu/patientist/logic/parser/AddCommandParser.java:61]: Small nit but could be removed if you simply declare patient outside this scope.

**Review by lukkesreysandeur — APPROVED** [review]
Code looks good to me, but I think that we may need to change the way statuses are added on to the patient. The way that statuses are being added on is more like a field, but a patient could potentially have more than one status, so it might be better for us to eventually update the status field in the patient class to a Set, similar to how Tags are implemented in AB3.

**Pull request merged**